cl/search/factories.py

```diff
@@ -45,6 +47,23 @@ class Meta:
     jurisdiction = FuzzyChoice(Court.JURISDICTIONS, getter=lambda c: c[0])
     in_use = True
 
+    @classmethod
+    def _create(cls, model_class, *args, **kwargs):
+        count = 1
+        while True:
+            try:
+                obj = model_class(*args, **kwargs)
+                obj.save()
+                return obj
+            except IntegrityError as exp:
+                print(f"Unexpected {exp=}, {type(exp)=}")
```

Comment:
Yeah, this method solves the issue I would just recommend using `logger.info() ` instead of `print()` for logging the error.